modules/gapi/perf/gpu/gapi_core_perf_tests_gpu.cpp

```diff
@@ -76,25 +79,28 @@ INSTANTIATE_TEST_CASE_P(DivPerfTestGPU, DivPerfTest,
                                 Values(cv::compile_args(CORE_GPU))));
 
 INSTANTIATE_TEST_CASE_P(DivCPerfTestGPU, DivCPerfTest,
-                        Combine(Values( szSmall128, szVGA, sz720p, sz1080p ),
+                        Combine(Values(AbsExact().to_compare_f()),
+                                Values( szSmall128, szVGA, sz720p, sz1080p ),
                                 Values( CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1 ),
                                 Values( -1, CV_8U, CV_16U, CV_32F ),
                                 Values(cv::compile_args(CORE_GPU))));
-
+//TODO: DivRC test doesn't work with (CV_16SC1, -1)
```

Comment:
`FIXIT:` would be better

Reply:
Did you create a ticket? If yes, could you please mention it here?

Reply:
That test case is working now. Removed that line